packages/strapi-design-system/src/v2/Breadcrumbs/Breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,97 @@
+<!--- Breadcrumbs.stories.mdx --->
+
+import { Meta, Story, Canvas } from '@storybook/addon-docs';
+import { ArgsTable } from '@storybook/addon-docs';
+import { Breadcrumbs } from './Breadcrumbs';
+import { Crumb } from './Crumb';
+import { CrumbLink } from './CrumbLink';
+import { Stack } from '../../Stack';
+import CollectionType from '@strapi/icons/CollectionType';
+
+<Meta title="Design System/Components/v2/Breadcrumbs" component={Breadcrumbs} />
+
+# Breadcrumbs
+
+Breadcrumbs are a list of links that inform users of their navigation.
```

Comment:
```suggestion
Breadcrumbs are a list of hierarchical links that inform users about their possible navigation path upwards.
```

Reply:
for the story's wording I pinged @maevalienard 
she will handle it 😊

Reply:
Well seems like it's done already so let's make profit of the time already spent :D I'll do it next time! :)